scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -542,16 +542,22 @@ public synchronized void update(TaskInfo taskInfo) {
         this.taskInfo = (TaskInfoImpl) taskInfo;
     }
 
-    /**
-     * @see org.ow2.proactive.scheduler.common.task.CommonAttribute#setMaxNumberOfExecution(int)
-     */
+
     @Override
     public void setMaxNumberOfExecution(int numberOfExecution) {
         super.setMaxNumberOfExecution(numberOfExecution);
         this.taskInfo.setNumberOfExecutionLeft(numberOfExecution);
         this.taskInfo.setNumberOfExecutionOnFailureLeft(maxNumberOfExecutionOnFailure);
     }
```

Comment:
Since this function does more than its title "setMaxNumberOfExecution", why not removing it, and replacing all setMaxNumberOfExecution(..) calls by setMaxNumberOfExecution(..), setNumberOfExecutionLeft(..) and setNumberOfExecutionOnFailureLeft(..)?